test/suite/functional/Generator/ServiceProvider/ServiceProviderPhp72.php

```diff
@@ -45,5 +47,11 @@ public function register(Container $container): void
         $container[FoodMapper::class] = static function () use ($container): FoodMapper {
             return new FoodMapper();
         };
+        $container[FindHumanByIdResponseBodyMapper::class] = static function () use ($container): FindHumanByIdResponseBodyMapper {
```

Comment:
smth is odd because it should've added the new serializer into (new BodySerializer())->add 🤔

Reply:
It is depends on requestBody. I added post request